frontend/src/router/index.js

```diff
@@ -225,16 +292,45 @@ export default function createRouter ({ store, userManager }) {
             },
             {
               path: ':name',
-              name: 'ShootItem',
-              component: ShootItemCards,
+              component: PlaceholderComponent,
               meta: {
                 namespaced: true,
                 projectScope: true,
                 title: 'Cluster Details',
                 toRouteName: 'ShootItem',
-                breadcrumbTextFn: routeParamName,
-                tabs: shootItemTabs
-              }
+                breadcrumbTextFn: routeParamName
+              },
+              children: [
```

Comment:
remove hierarchy

Reply:
hierarchy needed because of breadcrumb hierarchy
Project Clusters **-> my-shoot** -> Cluster Terminal